packages/catalyst_dapp_connector/lib/src/dapp_connector.dart

```diff
@@ -0,0 +1,63 @@
+@JS('cardano')
+import 'package:js/js.dart';
+import 'package:js/js_util.dart';
+import 'package:catalyst_dapp_connector/src/models/wallet_model.dart';
+import 'package:catalyst_dapp_connector/src/models/web_wallet_model.dart';
+import 'package:catalyst_dapp_connector/src/models/voting_credentials_model.dart';
+import 'package:catalyst_dapp_connector/src/models/cip62_api_model.dart';
+
+external WebWallet get flint;
+external WebWallet get nami;
+external WebWallet get ccvault;
+external WebWallet get gerowallet;
+external WebWallet get nufi;
+
+// TODO: Is there a way to not hardcode wallets?
```

Comment:
@kickloop `TODO` or `FIXME` are technical depts. We don't want to create any technical depts.
Only [flutter_style_todos](https://dart-lang.github.io/linter/lints/flutter_style_todos.html) are allowed if there is a good reason to do so.

Reply:
@minikin Sure. I understand. [flutter_style_todos](https://dart-lang.github.io/linter/lints/flutter_style_todos.html) would definitely work as well.

As for the tech debt, you are right and we would never leave it or pile up the tech debt :) As I was pointing out, we only use those TODO's during the development, so that we don't forget to take another look at the code in question.